app/services/current-session.js

```diff
@@ -7,11 +7,16 @@ import { findGroupByRole } from 'frontend-kaleidos/config/permissions';
 export default class CurrentSessionService extends Service {
   @service session;
   @service store;
+  @service impersonation;
 
   @tracked user;
-  @tracked role;
   @tracked organization;
   @tracked membership;
+  @tracked ownRole;
+
+  get impersonatedRole() {
+    return this.impersonation.role;
```

Comment:
Is this 'alias' necessary?  I would like to avoid exposing properties of other services through this service.